swap_meet/clothing.py

```diff
@@ -1,2 +1,13 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+
+class Clothing(Item):
+
+    def __init__(self, id=None, condition=0, fabric="Unknown"):
+        super().__init__(id, condition)
+        self.fabric = fabric
+
+    def __str__(self):
+        return (f"An object of type Clothing with id {self.id}."
+                f" It is made from {self.fabric} fabric.")
```

Comment:
We could combine these with a special character in Python strings! Using a `\n` will create a new line!

```python
        return (f"An object of type Clothing with id {self.id}.\nIt is made from {self.fabric} fabric.")
```